src/probnum/filtsmooth/_kalman_filter_smoother.py

```diff
@@ -0,0 +1,192 @@
+"""Convenience functions for filtering and smoothing."""
+
+import numpy as np
+
+from probnum import problems, randprocs, randvars, statespace
+from probnum.filtsmooth import gaussian
+
+__all__ = ["kalman_filter", "rauch_tung_striebel_smoother"]
+
+
+def kalman_filter(
+    observations, locations, F, L, H, R, m0, C0, prior_model="continuous"
```

Comment:
Do you think, using `drift`, `dispersion`, etc. would be more consistent and more clear than using `F`, `L`, etc.?

Reply:
More consistent, due to the attributes `driftmat` and `dispmat` etc. in the `Transition`s

Reply:
Yes, I was thinking this before. But perhaps it is easier to parse this interface for a user, for instance because other implementations of similar algorithms do this (e.g. matlab: https://uk.mathworks.com/help/control/ug/kalman-filtering.html). I can even change the letters to match the matlab style?!

Reply:
@schmidtjonathan this one has not been discussed yet :)

Reply:
Mh, personally, if I had to choose between F, L, etc. and the Matlab notation, I would choose the former. Additionally, if we start following Matlab conventions, we probably have to do it everywhere, right? 😅
From my side, we can just leave it as is, as long as it is well documented :)

Reply:
I agree. The Matlab comparison was just to motivate my choice of single-letter variables here :)